rfcs/0130-Implementing Checks API in tc-github while preserving Statuses API

```diff
@@ -0,0 +1,45 @@
+# RFC 131 - Implementing Checks API
```

Comment:
Quick formatting note -- this should be an `.md` file.  Probably `0131-implementing-checks-api.md`.